wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -19,6 +19,10 @@
 
 @import 'wagtailadmin/scss/fonts';
 
+#wagtail {
```

Comment:
This looks a bit vague to me, but I defer to your experience. Wouldn't `#wagtail-app-root` or similar, be more explicit and clear?